R/geom-contour.r

```diff
@@ -123,3 +124,11 @@ GeomContour <- ggproto("GeomContour", GeomPath,
     alpha = NA
   )
 )
+
+#' @rdname ggplot2-ggproto
+#' @format NULL
+#' @usage NULL
+#' @export
+#' @include geom-polygon.r
+GeomContourFilled <- ggproto("GeomContourFilled", GeomPolygon)
```

Comment:
What is the purpose of this? Do we have any precedence for creating ggproto aliases?

Reply:
This is needed to auto-generate documentation for aesthetics. We probably should do it more consistently across ggplot2. See here:
https://github.com/clauswilke/ggplot2/blob/743bad010638fdccaed4c0cfa0258e0879e1263c/R/geom-density2d.r#L9-L10

Reply:
Ah... that seems bad, but is not really the fault of this PR